.git-commit-template

```diff
@@ -0,0 +1,9 @@
+########50 characters############################
```

Comment:
Qué bueno. ¿Esto cómo se usa?

Reply:
Pues únicamente hay que hacer:
```
git commit
```
tras hacer un **git add** de algún fichero y se abre esta plantilla. Una vez que se escribe en la plantilla, se guarda y se cierra, se crea el commit con la información que tenga. Lo que esté comentado lo ignora (sirve de guía) y lo demás lo añade al commit. Lo uso como un "editor de commits". Para configurarlo solo hay que crear la plantilla y después, este comando:
```
git config commit.template ~/IV/Planner-IV/.git-commit-template

``` 
(En mi caso,  ***~/IV/Planner-IV/.git-commit-template***, es la ruta a la plantilla)
Esto lo hice a raíz de liarla tanto con el formato de los commits en los objetivos anteriores... :sweat_smile: así no se me pasa hacerlos lo mejor posible.
Lo vi en esta página:
https://medium.com/@hritik.jaiswal/how-to-write-a-good-commit-message-9d2d533b9052

Y me quedó más clara su configuración con esta otra página:
https://alex-wasik.medium.com/create-a-custom-git-commit-template-84468232a459